src/state/authentication/sagas.ts

```diff
@@ -1,4 +1,4 @@
-import SplashScreen from 'react-native-splash-screen';
+import RNBootSplash from "react-native-bootsplash";
```

Comment:
double `""` instead of a single quotation mark

Reply:
> As far as I understand someone with macOS should takeover it and finish implementation?

yes